Delete the deployment at the end of test, allow using upon-end parameter to "stop" or "keep". (#127)
opendatahub-io/ai-edge: pipelines/README.md

oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-bike-rental.yaml
```

> Check what pipeline was created and run and what applications started either in OpenShift Console in Pipelines > Pipelines,
> or with CLI find the names of the objects with

Reviewer: ```suggestion
Check what pipeline was created and what applications were started either using:
* OpenShift Console in Pipelines > Pipelines
  AND/OR
* CLI; find the names of the objects with:
```

Reviewer 2: ```suggestion
Check what pipeline was created and run, and what applications were started either in OpenShift Console in Pipelines > Pipelines,
or using the CLI to find the names of the objects by issuing the following commands
```
@piotrpdev I think your suggestion doesn't render correctly. I think it's ok to remove the `AND/OR` 

![image](https://github.com/opendatahub-io/ai-edge/assets/1930204/900af08d-c388-4cba-9ac7-3ed56180949d)

Author: I feel using the bullet points add too much significance to a step that is basically "if you are curious about what happened, these are some hints how to see the results". Or even: "this is what Jan does to make sure he understands what this steps was supposed to achieve".

Plus keeping that command block properly indented under that bullet point is a pain from the editing point of view.

Reviewer: > @piotrpdev I think your suggestion doesn't render correctly. I think it's ok to remove the `AND/OR`
> 
> ![image](https://user-images.githubusercontent.com/1930204/273145730-900af08d-c388-4cba-9ac7-3ed56180949d.png)

Fixed, needed spaces 😩

Reviewer: > I feel using the bullet points add too much significance to a step that is basically "if you are curious about what happened, these are some hints how to see the results". Or even: "this is what Jan does to make sure he understands what this steps was supposed to achieve".
> 
> Plus keeping that command block properly indented under that bullet point is a pain from the editing point of view.

It's hard to read though 😢 how about this then?

```suggestion
Check what pipeline was created and what applications were started either using the OpenShift Console in Pipelines > Pipelines, or with the CLI; find the names of the objects with:
```

Author: I went roughly with the wording suggested by @devguyio.

```bash
oc get pipeline
oc get pipelinerun
```
and then run `oc describe` on them.

If you add
```
   - name: upon-end
     value: keep
```

> to the PipelineRun's `params`, the applications that got created for testing will not get deleted and you can inspect them in OpenShift Console for example in Topology > select the application and its Resources and Pod logs, or with CLI find the names of the objects with

Reviewer: ```suggestion
to the PipelineRun's `params`, the applications that were created for testing will not be deleted and you can inspect them using:
* OpenShift Console in Topology > select the application and its Resources and Pod logs
  AND/OR
* CLI; find the names of the objects with:
```

Reviewer 2: @piotrpdev same comment as above

Reviewer: ```suggestion
to the PipelineRun's `params`, the applications that were created for testing will not be deleted and you can inspect them using:
* OpenShift Console in Topology > select the application and its Resources and Pod logs

  AND/OR
  
* CLI; find the names of the objects with:
```

or

```suggestion
to the PipelineRun's `params`, the applications that were created for testing will not be deleted and you can inspect them using the OpenShift Console in Topology > select the application and its Resources and Pod logs, or the CLI; find the names of the objects with:
```
